e2e_tests/tests/features/homepage.feature

```diff
@@ -8,9 +8,9 @@ Feature: Search and Purchase on Amazon
     Given User is on home page
     When User searches for "Snickers"
     Then The product "Snickers" is found and added to the cart
-    And Fail last step intentionally
 
-#   Scenario: Search for Skittles and add to cart
-#     Given User is on home page
-#     When User searches again for "Skittles"
-#     Then The product "Skittles" is successfully found and added to the cart
+  Scenario: Search for Skittles and add to cart
```

Comment:
Wasn't the task formulation: "A Cucumber-based descriptive test case to search for **the cheapest Snickers and Skittles**, add them to the basket, and verify the basket calculation and registration page redirection." ?

Reply:
Yes, but I assumed it would be better splitting into two distinct ones. rewrite it?